starknet/src/space/space.cairo

```diff
@@ -5,28 +5,71 @@ use sx::types::{
 
 #[starknet::interface]
 trait ISpace<TContractState> {
-    // State 
+    // -- View functions --
+
+    /// Returns the owner of the contract.
     fn owner(self: @TContractState) -> ContractAddress;
-    fn max_voting_duration(self: @TContractState) -> u32;
+
+    /// The voting delay to wait for between a proposal is created and the start of the voting period.
```

Comment:
The voting delay in seconds between when a proposal is created and the start of the voting period.

Reply:
3b561eb2d3c5ee7376822f546c0fe081368ddb63

Reply:
6f348d79f7d366db2741e52a881e17708515063c